packages/core/src/shader/GLProgram.ts

```diff
@@ -17,6 +17,19 @@ export class GLProgram
     public program: WebGLProgram;
     public uniformData: Dict<any>;
     public uniformGroups: Dict<any>;
+    /**
+     * a hash that stores where ubs are bound to on the program
+     */
+    public uniformBufferBindings: Dict<any>;
+    /**
+     * a hash for upload functions, we only generates new ones if they don't already exist
+     */
+    public uniformSync: Dict<any>;
+    /**
+     * a place where dirty ticks are stored for groups
```

Comment:
Can you clarify what "ticks" mean?

```suggestion
     * A place where dirty ticks are stored for groups.
```

Reply:
added a little more detail in the docs!